src/main/kotlin/org/wycliffeassociates/otter/common/data/rxmodel/Book.kt

```diff
@@ -0,0 +1,11 @@
+package org.wycliffeassociates.otter.common.data.rxmodel
+
+import io.reactivex.Observable
+
+data class Book(
```

Comment:
Should `Book` and `BookElement` have some shared supertype? Or, should `Book` be a `BookElement` (possibly with a null `audio` field, or possibly `HasAudio` should be its own interface, which Chunk and Chapter implement).

Or, for BookElement + BookElementWithAudio, see https://github.com/WycliffeAssociates/otter-common/pull/95